app/vmauth/README.md

```diff
@@ -331,6 +331,8 @@ See the docs at https://docs.victoriametrics.com/vmauth.html .
 
   -auth.config string
      Path to auth config. It can point either to local file or to http url. See https://docs.victoriametrics.com/vmauth.html for details on the format of this auth config
+  -brokenBeckendBackoff duration
```

Comment:
```suggestion
  -brokenBackendBackoff duration
```
Could you please fix this in other places as  well?

Reply:
Thanks! I fixed it all